utils.go

```diff
@@ -0,0 +1,84 @@
+// Copyright 2019 LINE Corporation
+//
+// LINE Corporation licenses this file to you under the Apache License,
+// version 2.0 (the "License"); you may not use this file except in compliance
+// with the License. You may obtain a copy of the License at:
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations
+// under the License.
+
+package centraldogma
+
+import (
+	"fmt"
+	"net/url"
+	"strconv"
+	"strings"
+)
+
+func setFromTo(v *url.Values, from, to string) {
```

Comment:
I guess it seems like these methods strongly bind specific parameters in this utility file. 
If we need a new parameter, we have to add a new method. 🤔
How about just adding a method which helps null check and reuses the existing logic? 😀

```go
func setUrlParamStringIfEmpty(v *url.Values, name, value string) {
	if len(value) != 0 {
		v.Set(name, value)
	}
}

func setUrlParamIntIfEmpty(v *url.Values, name string, value int) {
	if len(name) != 0 {
		v.Set(name, strconv.Itoa(value))
	}
}

// and 
setUrlParamStringIfEmpty(v, "from", from)
setUrlParamStringIfEmpty(v, "to", to)
setUrlParamIntIfEmpty(v, "maxCommits", maxCommits)
```

Reply:
:+1: Thank you very much. It's nice suggestion.

> if we need a new parameter, we have to add a new method
Yes, it's my intention. Since our param is vary, we need exactly function to add exact param name.

Reply:
I  address your comment. PTAL @ikhoon